packages/layout/src/components/cards/insurer/views/preview/preview.tsx

```diff
@@ -35,6 +38,14 @@ export const Preview: React.FC<any> = React.memo(() => {
 							country: insurer.address.country,
 						}}
 					/>
+
+					{/* Companies House Number: display only	 */}
+					{insurer.companiesHouseNumber && (
+						<CompaniesHouseNumber
+							heading={i18n.preview.buttons.four}
+							houseNumber={insurer.companiesHouseNumber}
```

Comment:
`houseNumber` is a bit of a confusing property name. You'll find a house number outside your front door, but Companies House is the name of an organisation. It's never known as just "house".